sbnana/CAFAna/Core/Tree.cxx

```diff
@@ -1280,8 +1283,11 @@ namespace ana
       // Save the weights
       for ( unsigned int idxBranchWeight=0; idxBranchWeight<fOrderedBranchWeightNames.size(); ++idxBranchWeight ) {
         const int NSigmas = fNWeightsExpected.at( fOrderedBranchWeightNames.at(idxBranchWeight) );
+        const double lo = fNSigmasLo.at( fOrderedBranchWeightNames.at(idxBranchWeight) );
+        const double hi = fNSigmasHi.at( fOrderedBranchWeightNames.at(idxBranchWeight) );
         for ( unsigned int idxWt = 0; idxWt < (unsigned int)NSigmas; ++idxWt ) {
           weights[ fOrderedBranchWeightNames.at(idxBranchWeight) ].push_back(fBranchWeightEntries.at( fOrderedBranchWeightNames.at(idxBranchWeight) ).at(idxEntry).at(idxWt));
+          knob_vals[ fOrderedBranchWeightNames.at(idxBranchWeight) ].push_back(lo + (hi-lo)*(double)idxWt/(NSigmas-1));
```

Comment:
In the way the sigmas are current implemented isn't this just equivalent to 
```
double(lo + idxWt)
```
since I think `hi-lo` will always be `NSigmas-1` here, but I may be not thinking of something?

However, @jedori0228 has some nice updates in another PR that changes the way that the NSigmas are required to work, enabling more customization. Jaesung, could you bring this into your branch and update the logic here to match your updates?

If I recall correctly, in your PR they are already saved as a vector of sigmas, so I think it might be pretty easy? Let me know if not though!

Reply:
That is included in #113 (commit: https://github.com/SBNSoftware/sbnana/commit/f92120a0b7d2d407f9d95255dbe206e1b05e8942). @jacoblarkin Would you take a look at this one and cherry-pick?